components/badge.css

```diff
@@ -0,0 +1,70 @@
+.badge-container {
+    position: relative;
+    width: fit-content;
+    box-sizing: border-box;
+}
+
+.status-badge {
+    height: 1.1rem;
+    width: 1.1rem;
+    border-radius: 50%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+    position: absolute;
+    font-size: 0.8em;
+    color: white;
+    padding: 0.5rem;
+}
```

Comment:
check colors , use color variables

if the value is less than 10 px you can use px instead of rems.